quark-secrets-log.txt

```diff
@@ -0,0 +1,275 @@
+2021-02-04T13:37:45.405Z	[34mINFO[0m	cmd/kubeconfig.go:16	Using in-cluster kube config
```

Comment:
also mistake?

Reply:
Very likely yes. Will remove.

It seems that I was in too much of a hurry when I made this :(

Reply:
Done with commit 0d380cfe9c5b24cdbc7eec2615976d3c88af3723 (rebase).